src/sentry/api/endpoints/organization_events.py

```diff
@@ -43,6 +43,10 @@
     DiscoverSavedQueryTypes.TRANSACTION_LIKE: get_dataset("discover"),
     DiscoverSavedQueryTypes.ERROR_EVENTS: get_dataset("errors"),
 }
+DASHBOARDS_DATASET_MAP = {
+    DashboardWidgetTypes.TRANSACTION_LIKE: get_dataset("metricsEnhanced"),
```

Comment:
okay so this is fine for now because the codepath that currently runs this is scoped to metricsEnhanced dataset but I realized we don't attempt to split the dataset for this codepath 
https://github.com/getsentry/sentry/blob/master/src/sentry/api/endpoints/organization_events.py#L590-L591

this would have to hit the `transactions` dataset instead of `metricsEnhanced`